server/auth/types/saml/routes.ts

```diff
@@ -81,7 +83,8 @@ export class SamlAuthRoutes {
 
     this.router.post(
       {
-        path: '/_plugins/_security/saml/acs',
+        // need to change to /_opendistro to execute the tests.
+        path: '/_opendistro/_security/saml/acs',
```

Comment:
Is there a way you can test both endpoints or just the new one?

Reply:
we can ignore this, since you had already merged this change